sdcm/provision/common/configuration_script.py

```diff
@@ -89,6 +91,15 @@ def _script_body(self) -> str:
             script += 'systemctl start sshd || true\n'
         return script
 
+    @staticmethod
+    def _enable_legacy_rsa_keys():
+        """
+        Add support for RSA-1 ssh keys in sshd config.
+        """
+        script = configure_legacy_rsa_keys_for_sshd()
+        script += restart_sshd_service()
```

Comment:
why not adding the whole thing into `configure_sshd_script` ? why do we need multiple restarts of the sshd ?